### ISSUE TITLE
browsh output full of question marks + bottons on YouTube don't show up

### ISSUE THREAD

**Issue by iComputeDaily** [browsh-org/browsh#273]
![gscreenshot_2019-07-13-145323](https://user-images.githubusercontent.com/52219059/61175524-0bdd6880-a57f-11e9-9ec3-768b8472b192.png)

I can't figure out why the output of Browsh is full of question marks, as I am definitely using Unicode, and a terminal emulator that has truecolor support (st with path set to st-direct)

Additionally, if I go to youtube my cpu usage gose to only 20% and the video does not actually play neither can I press the play or fullscreen bottons, as they appear to bo offscreen.

I get similar results with konsole, and even worse results(more question marks) with tty.

I installed browsh with the browsh-bin AUR package, and have no idea why these problems are occurring, could anyone point me in the right direction?

**Comment by iComputeDaily** [comment]
I have also just realized I can't switch tabs

**Comment by tombh** [comment]
Hmm, what happens if you switch the text visibility mode with F6?

**Comment by iComputeDaily** [comment]
I can't tell any difference from my original screenshot when I turn visibility mode on; What is it supposed to change?

**Comment by tombh** [comment]
Ok, it's not that then.

And what happens when you do `ssh brow.sh`?

**Comment by iComputeDaily** [comment]
it gets even more messed up 
![image](https://user-images.githubusercontent.com/52219059/63522556-95f5e680-c4c6-11e9-8e17-07c4646d4785.png)

**Comment by iComputeDaily** [comment]
(it got more messed up when I scrolled; scrolling dose not produce this on the local instance though)

**Comment by tombh** [comment]
Then this proves that it is a problem with your terminal. Which one is it?

**Comment by iComputeDaily** [comment]
the original screenshot was taken with st, and the latter with konsole

**Comment by tombh** [comment]
Oh they both should be fine. And do you know exactly which font is being used?

**Comment by iComputeDaily** [comment]
After digging through the settings in Konsole I found that I was using windows-1258 encoding for some reason...
After switching it to utf-8 the ssh demo worked fine, but the local instance still does not.
I also found out I am using the font "Hack 9pt".

**Comment by iComputeDaily** [comment]
No, the local instance did not look any different even with UTF-8 encoding

**Comment by adithyaov** [comment]
Hi, I'm facing the same issue on `st` and `xterm`. Is there a way to
debug/fix this? I can help with the fix if I have some pointes.

PS: Thank you for the amazing work!

Best,
Adithya
